pro/cypress/support/commands.ts

```diff
@@ -38,4 +38,30 @@
 
 // TODO Albéric: can be re-added when this issue is solved: https://github.com/testing-library/cypress-testing-library/issues/252
 // import '@testing-library/cypress/add-commands'
+
+Cypress.Commands.add('login', (email: string, password: string) => {
+  return cy
+    .visit('http://localhost:3001/connexion')
+    .get('#email')
+    .type(email)
+    .get('#password')
+    .type(password)
+    .get('button[type=submit]')
+    .click()
+})
+
+Cypress.Commands.add('setFeatureFlags', (features: Feature[]) => {
+  return cy.request({
+    method: 'PATCH',
+    url: 'http://localhost:5001/native/v1/features',
```

Comment:
J'ai poussé mon commit de modification de l'URL : 

```suggestion
    url: 'http://localhost:5001/testing/features',
```